.nvmrc

```diff
@@ -0,0 +1 @@
+13
```

Comment:
Just helps when working locally that we all use the same node version. Plus we can use it later on CI.